Source/WebCore/page/LocalFrameView.cpp

```diff
@@ -3343,7 +3343,7 @@ void LocalFrameView::setNeedsLayoutAfterViewConfigurationChange()
 void LocalFrameView::setNeedsCompositingConfigurationUpdate()
 {
     RenderView* renderView = this->renderView();
-    if (renderView && renderView->usesCompositing()) {
+    if (renderView && renderView && renderView->usesCompositing()) {
```

Comment:
```suggestion
    if (renderView && renderView->usesCompositing()) {
```
also, line above could use `auto*`